src/PerCapita.less

```diff
@@ -1,7 +1,10 @@
 @import (less) "base.less";
 
 .per-capita {
-  select {
-    height: 34px;
+  button {
+    color: @black;
+    &.deselected {
+      background-color: @pacific-40;
```

Comment:
It looks like this color may be slightly off compared to the date range selector:
![Screen Shot 2020-03-26 at 1 09 50 PM](https://user-images.githubusercontent.com/1028659/77675273-1c920b00-6f63-11ea-9834-aa0a109de00d.png)

The contrast difference with black text is subtle but makes it difficult to read.

Minor fix if you can get to it in this PR. Thanks!

Reply:
changed.